### PR TITLE
move 3 extensions to prod

### PR DESCRIPTION
**Pull request by RhinosF1**
+1 from me on all 3

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -426,8 +426,7 @@
 	],
 	//CommonsMetadata
 	'wgCommonsMetadataForceRecalculate' => [
-		'default' => false,
-		'test1wiki' => true,
+		'default' => true,
```
Comment (paladox) [inline LocalSettings.php]: Why is this being enabled by default?

File: ManageWikiExtensions.php [hunk 2]
```diff
@@ -1665,12 +1657,7 @@
 			'linkPage' => 'https://www.mediawiki.org/wiki/Extension:TextExtracts',
 			'var' => 'wmgUseTextExtracts',
 			'conflicts' => false,
-			'requires' => [
-				'permissions' => [
-					'managewiki-restricted',
-				],
-			],
-		],
+			'requires' => [],
```
Comment (paladox) [inline ManageWikiExtensions.php]: Syntax error

File: LocalSettings.php (hunk 1 above)
Comment (RhinosF1) [reply LocalSettings.php]: As it allows remote files (from miraheze/wikimedia) commons to be used if enabled for this extension. As far as I can tell, those images wouldn't work without it.

Comment (paladox) [reply LocalSettings.php]: This should be made configurable in MW and disabled by default.

Based on me reading https://www.mediawiki.org/wiki/Extension:CommonsMetadata/en there may be a performance hit with this enabled. Secondly this you can already use remote files.

What the extension does is it pulls meta data.

**Comment by RhinosF1** [comment]
Reverting while I fix i18n to avoid issues

**Pull request merged**